acceptancetests/jujupy/k8s_provider/base.py

```diff
@@ -169,6 +170,8 @@ def add_k8s(self, is_local=False, juju_home=None):
                 '--local',
                 '--cluster-name', self.kubeconfig_cluster_name,
             )
+            # use this local cloud to bootstrap later.
+            self.bs_manager.client.env._cloud_name = self.cloud_name
```

Comment:
This seems a bit like reaching deep into the internals of client.env - maybe add a method to it to allow setting the cloud name? Not a biggie, but might save some confusion later on.

Reply:
right, added set_cloud_name method in `JujuData`.